klippy/extras/tmc2240.py

```diff
@@ -0,0 +1,397 @@
+# TMC2240 configuration
+#
+# Copyright (C) 2018-2023  Kevin O'Connor <kevin@koconnor.net>
+# Copyright (C) 2023  Alex Voinea <voinea.dragos.alexandru@gmail.com>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+import math, logging
+from . import bus, tmc, tmc2130
+
+TMC_FREQUENCY=12500000.
+
+Registers = {
+    "GCONF":            0x00,
+    "GSTAT":            0x01,
+    "IFCNT":            0x02,
+    "SLAVECONF":        0x03,
```

Comment:
This register was recently renamed by Trinamic in the rev1 tmc2240 datasheet to `NODECONF`. It seems they renamed all "slave" terms in the datasheet to "node". Should we update this register name as well or not? The other TMC drivers still call it `SLAVECONF` in the datasheet, but that might just be because they haven't updated the documents yet.